app/models/repository_host/raw_upstream_data_converter.rb

```diff
@@ -0,0 +1,79 @@
+# frozen_string_literal: true
+
+class RepositoryHost::RawUpstreamDataConverter
+  def self.convert_from_github_api(upstream_repository_data_hash)
```

Comment:
Should the error handling get standardized in here too? Saw a few `rescue *self::IGNORABLE_EXCEPTIONS nil` in the code. 

Alternatively, now that this is written out, it looks like there's some pretty tight coupling w/ the api return objects, so the convert_ methods could be moved to their respective methods in the repository_host/<host>.rb files, as there doesn't seem to be a ton that's similar between the convert methods besides the object that's tying everything together, unless stuff like error handling is getting standardized. I was thinking there'd be more similarities between the objects being converted that there could be more extracted out and shared during the conversion process, like defaulting values/data validations/logging/etc.